homeassistant/components/vesync/const.py

```diff
@@ -32,6 +37,9 @@
 VeSyncHumidifierDevice = VeSyncHumid200300S | VeSyncSuperior6000S
 """Humidifier device types"""
 
+VeSyncFanDevice = VeSyncAirBypass | VeSyncAir131
+"""Humidifier device types"""
```

Comment:
This is incorrect.

Reply:
Removed. I feel like that was for fan refactor and had comment errors.